api/Window.json

```diff
@@ -3870,10 +3870,10 @@
               "version_added": "47"
             },
             "safari": {
-              "version_added": null
+              "version_added": "4"
```

Comment:
This matches [`postMessage()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage#browser_compatibility) and https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/SafariJSProgTopics/Cross-documentmessaging.html says "This feature is supported in Safari 4.0 and later" 👍